internal/neofs/neofs.go

```diff
@@ -259,6 +259,9 @@ func (x *NeoFS) CreateObject(ctx context.Context, prm layer.PrmObjectCreate) (oi
 		var lock object.Lock
 		lock.WriteMembers(prm.Locks)
 		obj.WriteLock(lock)
+
+		// we can't have locks and payload at the same time.
```

Comment:
I'd check for this and return an error. Otherwise it'd take some to debug if anyone accidentally to push both into `prm`.

Reply:
Updated